keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -41,21 +45,6 @@ class ResourceActuator(
 
   suspend fun <T : ResourceSpec> checkResource(resource: Resource<T>) {
     val id = resource.id
-    val response = vetoEnforcer.canCheck(id)
```

Comment:
This covered application level vetoes, right? I think that particular veto type should be special cased and still eval'd here, so it can short circuit diff generation.

Reply:
Do you think it even belongs in a veto? I'm happy making it another type of thing.
Another option is we could have two places where we evaluate vetos, and have each veto choose where it's evaluated...

Reply:
Adding a todo to fix this - I don't want to make this PR any bigger.

Reply:
At the same time it might make sense to be able to provide users the context that there is a diff in their resource even though we're not going to resolve it.

Reply:
At this moment I don't think that's that useful, especially because we don't have a way to surface that anywhere